src/frost.rs

```diff
@@ -388,12 +394,12 @@ pub struct SigningPackage {
     /// Message which each participant will sign.
     ///
     /// Each signer should perform protocol-specific verification on the message.
-    pub message: &'static [u8],
+    pub message: Vec<u8>,
```

Comment:
You could add a named lifetime to avoid copying, but using a `Vec` should work for now.

```
pub struct SigningPackage<'msg> {
    pub message: &'msg [u8],
}
```

Reply:
Thanks, lets leave it as a `Vec` by now and change it if needed.